src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.queue.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
```

Comment:
I think @vzhukovsky was asking why you are not setting the `called` property just like [below](https://github.com/aol/aol-hb.js/pull/16/files#diff-e8024450d0d2f5327ea857a09562e878R51).